src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +78,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
Note that the coding style specifies that there shouldn't be indentation for case clauses.